plugins/eksporter.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: eksporter
+spec:
+  version: "v1.0.0"
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux, windows]}
```

Comment:
This line is a bit long, can you unfold {} into separate lines:

```
- key: os
  operator: In
  values:
  - darwin
  - linux
  - windows
```

I think this is the first plugin we ever have that supports all 3 OSes.

Reply:
Do we even need a selector when I target all 3 OSes?

I'll update the code anyway, as it looks better on separated lines.

Reply:
That’s a good point.
I think this used to work like:

   matchLabels: {}

But we recently introduced tighter validation code I believe. We might relax it in the future. Well, at least for now it’s a one-off operation.

Reply:
Indeed we did prevent this: https://github.com/kubernetes-sigs/krew/blob/8d148b5be03eb1a6611378ddd78d412cd3a9b5da/pkg/index/validation/validate.go#L162

Reply:
so I think matchLabels:{} might be worming today but it will stop working in the next versions of krew.